src/PluginManager.js

```diff
@@ -75,7 +75,17 @@ export const regexRules = [
         note: "The full-width characters '{matchString}' are not allowed in the target string. Use ASCII symbols instead.",
         regexps: [ "[\\uFF01|\\uFF05|\\uFF1F]+" ],
         link: "https://github.com/ilib-js/i18nlint/blob/main/docs/resource-no-fullwidth-punctuation-subset.md"
-    }
+    },
+    {
+        type: "resource-target",
+        name: "resource-no-double-byte-space",
+        description: "Ensure that the target does not contain double-byte space characters.",
+        note: "Double-byte space characters should not be used in the target string. Use ASCII symbols instead.",
+        // per https://en.wikipedia.org/wiki/Whitespace_character
+        regexps: [ "[\\u1680|\\u2000|\\u2001|\\u2002|\\u2003|\\u2004|\\u2005|\\u2006|\\u2007|\\u2008|\\u2009|\\u200A|\\u2028|\\u2029|\\u202F|\\u205F|\\u3000]+" ],
```

Comment:
Are the "or" vertical bars needed in a character class? Can it just be `[\\u1680\\u2000 ...`  ?

Reply:
Actually it shouldn't be there, fixed in f218e3c